crates/burn-jit/src/kernel/reduce/naive/argmax.rs

```diff
@@ -1,31 +1,29 @@
+use cubecl::prelude::*;
+
+use crate::{kernel::reduce::Argmax, JitElement};
+
 use super::base::ReduceDimNaive;
-use crate::kernel::reduce::Argmax;
-use cubecl::cube;
-use cubecl::frontend::{Tensor, ABSOLUTE_POS};
-use cubecl::prelude::{Cast, Numeric};
 
-#[allow(clippy::extra_unused_type_parameters)]
 #[cube]
-impl<EI: Numeric> ReduceDimNaive<EI> for Argmax {
-    type Accumulator = (f32, u32);
+impl<EI: JitElement> ReduceDimNaive<EI> for Argmax {
+    type Accumulator = (EI, u32);
 
-    fn initialize_naive() -> (f32, u32) {
+    fn initialize_naive() -> Self::Accumulator {
         // TODO: switch to using f32::NEG_INFINITY when it's supported: https://github.com/tracel-ai/cubecl/issues/68
```

Comment:
Is this comment still relevant?

Reply:
actually not sure, I tried replacing it with `NEG_INFINITY` at one point but it failed because it doesn't get printed properly in the generated code

Reply:
Looks like the issue is still unresolved due to issues with wgsl